Emby.Naming/Video/ExtraResolver.cs

```diff
@@ -76,7 +60,7 @@ public ExtraResult GetExtraInfo(string path)
                 {
                     var filename = Path.GetFileName(path);
 
-                    var regex = new Regex(rule.Token, RegexOptions.IgnoreCase);
+                    var regex = new Regex(rule.Token, RegexOptions.IgnoreCase | RegexOptions.Compiled);
```

Comment:
Would be better to have the `Regex` cached in the `rule` so we don't keep `new`ing them up at `IsMatch` time.

Reply:
That is true. The rules are due for a general refactor, which is out of scope of this (already big) PR.

Reply:
Just to follow up, there's currently no Regex-based ExtraTypeRule implemented